src/frontend/js/media-management/component/edit-sidebar.tsx

```diff
@@ -305,6 +322,31 @@ export default function EditSidebar({
           )}
         </div>
       </form>
+
+      {/* Hidden form for file replacement */}
+      <form
+        onSubmit={submitForm}
+        action={apiEndpoints.replaceFile}
+        class="hidden"
+      >
+        <input name="id" type="hidden" value={file.id} />
+        <input
+          id="replace-file-input"
+          name="file"
+          type="file"
+          maxLength={255}
+          accept={changedFile.type}
```

Comment:
```suggestion
          accept={file.type}
```
`changedFile` is a "special" buffer for changes to the name or alt tags, so I think it's more canonical to use just `file` here.